store/postgres/src/chain_store.rs

```diff
@@ -1877,6 +1954,42 @@ mod recent_blocks_cache {
             }
         }
 
+        fn record_cache_hit(&self) {
+            self.metrics
+                .chain_head_cache_hits
+                .get_metric_with_label_values(&[&self.network])
+                .unwrap()
+                .inc();
+        }
+
+        fn record_cache_miss(&self) {
+            self.metrics
+                .chain_head_cache_misses
+                .get_metric_with_label_values(&[&self.network])
+                .unwrap()
+                .inc();
+        }
```

Comment:
You could move `fn record_cache_hit` and `fn record_cache_miss` into `ChainStoreMetrics`.

Reply:
:+1: fixed in https://github.com/graphprotocol/graph-node/pull/4440/commits/efbf522284bd3cc705939d4419ed7343d77e9564.